lib/resources/r4/insurance_plan.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: InsurancePlan
+field_name_base_url: http://hl7.org/fhir/R4/insuranceplan-definitions.html#InsurancePlan
+fields:
+
+  - name: status
+    type: code
+    description: The current state of the health insurance product.
+    action: terminology
```

Comment:
do we not need any url for the status?

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/809/commits/f63da01cf6f9be92d7efbf579e6744e0b1ae0a89#diff-fc7acc4fa1e5e119e6b5e5a648f48327c4079574d8999f0b95fc28d8e463f2d4R10